packages/lib/gpcircuits/src/util.ts

```diff
@@ -56,7 +56,62 @@ export function gpcArtifactPaths(
 }
 
 /**
- * Returns an array which is a copy of `inputArray` extended to `totalLength`,
+ * Splits an array `arr` into chunks of size
+ * `n` in order. If `arr.length` is not a multiple of `n`,
+ * then the last chunk will be of length `arr.length % n`.
+ */
+export function toChunks<A>(arr: A[], n: number): A[][] {
+  const chunks: A[][] = [[]];
```

Comment:
Doesn't matter for the current usage, but it's a somewhat weird edge case that if the input array is empty this returns [[]] (1 chunk of size 0) rather than [] (no chunks).